pylint/reporters/ureports/nodes.py

```diff
@@ -14,30 +14,33 @@
 
 A micro report is a tree of layout and content objects.
 """
+from typing import Any, List, Optional, Tuple, Union
+
+from pylint.reporters.ureports.text_writer import TextWriter
 
 
 class VNode:
-    def __init__(self, nid=None):
+    def __init__(self, nid: Optional[Any] = None) -> None:
         self.id = nid
```

Comment:
Is `self.id` ever used? Searching for `.id` in the repository did not return anything that caught my eye. This might allow us to remove the Any typing from `nid`.

Reply:
I did not see it either. Report id is the closest to it.

Reply:
Probably remove then? Or do you want to ask Marc for input?

Reply:
I'm going to rebase on 4973 first, so I'm waiting for it to be merged.

Reply:
Probably able to remove now 😄